docs/gettingstarted/index.rst

```diff
@@ -159,6 +159,9 @@ Getting started with Flocker
          .. version-code-block:: console
 
             you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+              curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
```

Comment:
One thing which worries me, I should have done it in the branch which introduced the ``Vagrantfile`` above is that these file only exist because of a file which happens to be created elsewhere. I think add a comment over the downloads to mention that they are used here.